pages/studentPortal/index.tsx

```diff
@@ -67,6 +68,15 @@ export default function StudentPortalPage() {
 
   return (
     <div className="flex flex-col w-full px-4 pb-4 sm:px-8 sm:pb-8 ">
+      <DialogComponent
```

Comment:
let's take this component out for now until we have logic to call it conditionally (i.e you're first time on skillify, trial is over etc.)